app/assets/javascripts/map.js

```diff
@@ -4,16 +4,26 @@ $(function() {
 
   var createMap = function(latitude, longitude){
     L.mapbox.accessToken = window.MAPBOX_ACCESS_TOKEN;
-    map = L.mapbox.map('map', window.MAPBOX_USER_NAME).
+    map = L.mapbox.map('map', window.MAPBOX_USER_NAME, { zoomControl: false }).
       setView([latitude, longitude], 14);
   };
 
   var createMarker = function(court) {
     var marker = L.marker([ court.latitude, court.longitude], {
       icon: L.mapbox.marker.icon({
-        'marker-color': '#9c89cc'
+        'marker-color': '#9c89cc',
       })
-    }).addTo(map);
+    })
+    .addTo(map);
+    marker.on("click", function(){
+      $(".information-box").html(
+          "<div>"+court.name+"</div>" +
+          "<div>"+court.address+"</div>" +
+          "<div>Average Elo: "+court.average_elo+"</div>"+
+          "<a href=/courts/"+court.id+">Go to Court</a>"
```

Comment:
Not necessarily needed now, but could be nice to use something like [handlebars](http://handlebarsjs.com/) for the HTML templating.